dynamic-demo-plugin/console-extensions.json

```diff
@@ -349,5 +349,47 @@
       "name": "Modal Launchers",
       "href": "/test-modal"
     }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": false,
+      "path": ["/example-namespaced-page/ns/:ns", "/example-namespaced-page/all-namespaces"],
+      "component": {
+        "$codeRef": "exampleNamespacedPage.ExampleNamespacedPage"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "namespaced": true,
+      "id": "admin-example-namespaced-page",
+      "perspective": "admin",
+      "section": "admin-demo-section",
+      "name": "%plugin__console-demo-plugin~Example Namespaced Page%",
+      "href": "/example-namespaced-page"
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "namespaced": true,
+      "id": "dev-example-namespaced-page",
+      "perspective": "dev",
+      "section": "dev-demo-section",
+      "name": "%plugin__console-demo-plugin~Example Namespaced Page%",
+      "href": "/example-namespaced-page"
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "namespaced": true,
+      "id": "demo-example-namespaced-page",
+      "perspective": "demo",
+      "name": "%plugin__console-demo-plugin~Example Namespaced Page%",
+      "href": "/example-namespaced-page"
+    }
```

Comment:
any reason to have the page on all perspectives ? isnt demo perspective enough ?

Reply:
I wasn't sure. Do you know why some are on all three perspectives and others are not? Should all of them be on just the demo perspective?

Reply:
I dont know why others..but this one can be just in demo perspective IMO

Reply:
I already updated it to only be in the demo perspective. 😃